commonui/build.gradle

```diff
@@ -11,6 +11,7 @@ apply from: "$rootProject.projectDir/android-library.gradle"
 apply from: "$rootProject.projectDir/publish.gradle"
 
 dependencies {
+    api "com.hyperwallet.android:insight-sdk:$hyperwalletInsightVersion"
```

Comment:
I think we should use implementation here instead. If I understood correctly, the HyperwalletInsight class should be the entry point that the ui should use to interact with the Insight sdk. Leaving it as api would allow directly usages of the insight sdk classes by ui modules